cmd/kubelet/app/server.go

```diff
@@ -201,6 +204,8 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:     defaultRootDir,
 		SyncFrequency:     10 * time.Second,
 		SystemContainer:   "",
+		KubeApiQps:        20.0,
```

Comment:
I am ok with current default setting. But if I remembered correctly, kubelet's default value is 5, not 20.